### PR TITLE
Add AWS Bracket

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/amazon-braket-default-simulator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/amazon-braket-default-simulator-{{ version }}.tar.gz
+  sha256: 9880d8044e49415a6e4c862314a23277915b7de1c6a236992df255f31c2f550f
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (BastianZim) [inline recipes/amazon-braket-default-simulator/meta.yaml:15]: Not `noarch` due to https://github.com/aws/amazon-braket-default-simulator-python/blob/8b3e7478abca239eb6fd68d4ce4c41cee6bfd49d/setup.py#L30-L36

**Comment by BastianZim** [comment]
@conda-forge/help-python Would be ready for review, thanks!

File: recipes/amazon-braket-default-simulator/meta.yaml (hunk 1 above)
Comment (wolfv) [reply recipes/amazon-braket-default-simulator/meta.yaml:15]: I am unsure that this is a problem... let me check.

Comment (wolfv) [reply recipes/amazon-braket-default-simulator/meta.yaml:15]: The conda-forge knowledge base says:

> Note Only console_scripts entry points have to be listed in meta.yaml. Other entry points do not conflict with noarch and therefore do not require extra treatment.

Comment (BastianZim) [reply recipes/amazon-braket-default-simulator/meta.yaml:15]: Oh, yeah you're right, I completely misremembered that. Thanks!

**Comment by wolfv** [comment]
I think the indentation is off, unfortunately.

**Comment by BastianZim** [comment]
Well, I clearly need some coffee. Thanks!

**Comment by BastianZim** [comment]
By the way, with or-tools. Are you going to create the patch or for whom should I wait? Just so that I can make sure that I'm not waiting while the fix is already released.

**Comment by BastianZim** [comment]
Thank you!

**Comment by wolfv** [comment]
> By the way, with or-tools. Are you going to create the patch or for whom should I wait? Just so that I can make sure that I'm not waiting while the fix is already released.
> 
> Edit: I'm already following [conda-forge/coin-or-utils-feedstock](https://github.com/conda-forge/coin-or-utils-feedstock) but just wanted to check.

If you have the time to do it, that would be awesome. Sorry, I am a bit swamped these days :/

**Comment by BastianZim** [comment]
I have to admit I don't know where to even start with that one... 😅  I'll see what I can do.

**Pull request merged**